io/net/htx_nic_devices.py.data/htx_nic_devices.yaml

```diff
@@ -11,4 +11,4 @@ peer_ips: ""
 net_ids: ""
 # time limit in minutes
 time_limit: 2
-htx_rpm: ""
+htx_rpm: "http://ausgsa.ibm.com//gsa/ausgsa/projects/h/htx/public_html/htxonly/htxrhel8-544-LE.ppc64le.rpm"
```

Comment:
@vaishnavibhat seems me this is not accessible via public network , may be this can be target as internal patch